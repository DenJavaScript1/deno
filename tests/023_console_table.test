args: tests/023_console_table.ts --reload
output: tests/023_console_table.ts.out
