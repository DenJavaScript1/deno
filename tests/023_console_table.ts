console.table([0, 1, 2, 3, 4]);
console.table({
  name: "Jonh",
  surname: "Doe",
  age: "32",
  job: "Coder"
});
