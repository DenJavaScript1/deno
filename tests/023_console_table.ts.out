┌─────────┬────────┐
│ (index) │ Values │
├─────────┼────────┤
│    0    │   0    │
│    1    │   1    │
│    2    │   2    │
│    3    │   3    │
│    4    │   4    │
└─────────┴────────┘
┌─────────┬─────────┐
│ (index) │ Values  │
├─────────┼─────────┤
│  name   │ "Jonh"  │
│ surname │  "Doe"  │
│   age   │  "32"   │
│   job   │ "Coder" │
└─────────┴─────────┘
